<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Parallel test suite" duration-ms="21" started-at="2017-04-08T16:13:24Z" finished-at="2017-04-08T16:13:24Z">
    <groups>
      <group name="runAll">
        <method signature="SimpleTest.testCreateFileNegative()[pri:0, instance:SimpleTest@3567135c]" name="testCreateFileNegative" class="SimpleTest"/>
      </group> <!-- runAll -->
      <group name="CreateFileNegative">
        <method signature="SimpleTest.testCreateFileNegative()[pri:0, instance:SimpleTest@3567135c]" name="testCreateFileNegative" class="SimpleTest"/>
      </group> <!-- CreateFileNegative -->
    </groups>
    <test name="Test1" duration-ms="21" started-at="2017-04-08T16:13:24Z" finished-at="2017-04-08T16:13:24Z">
      <class name="SimpleTest">
        <test-method status="PASS" signature="testCreateFileNegative()[pri:0, instance:SimpleTest@3567135c]" name="testCreateFileNegative" duration-ms="9" started-at="2017-04-08T19:13:24Z" finished-at="2017-04-08T19:13:24Z">
          <exception class="java.io.IOException">
            <message>
              <![CDATA[No such file or directory]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: No such file or directory
	at java.io.UnixFileSystem.createFileExclusively(Native Method)
	at java.io.File.createNewFile(File.java:1012)
	at FileManager.createFile(FileManager.java:33)
	at SimpleTest.testCreateFileNegative(SimpleTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateFileNegative -->
      </class> <!-- SimpleTest -->
    </test> <!-- Test1 -->
  </suite> <!-- Parallel test suite -->
</testng-results>
